octobot_trading/exchanges/connectors/simulator/exchange_simulator_connector.py

```diff
@@ -150,13 +150,17 @@ def get_available_time_frames(self):
     def get_backtesting_data_files(self):
         return [backtesting_api.get_data_file_path(importer) for importer in self.exchange_importers]
 
-    def get_market_status(self, symbol, price_example=0, with_fixer=True):
+    def get_market_status(self, symbol, price_example=0, with_fixer=True, return_real_status_confirm=False):
+        # return_real_status_confirm is specific to simulator and returns True if the market status has been found
+        # in real market status cache
         if self._forced_market_statuses:
             try:
                 if with_fixer:
                     return util.ExchangeMarketStatusFixer(
                         self._forced_market_statuses[symbol], price_example
                     ).market_status
+                if return_real_status_confirm:
+                    return self._forced_market_statuses[symbol], True
                 return self._forced_market_statuses[symbol]
```

Comment:
Is it safe to have a tuple as return and sometimes not a tuple?

Reply:
up